bounties/npm/create-git/1/README.md

```diff
@@ -0,0 +1,31 @@
+## Details:
+I would like to report a `RCE` issue in the `create-git` module.
+It allows to `execute remote commands on the victim's PC`
+
+## Module
+* **module name:** `create-git`
+* **version:** `0.0.4`
+* **npm page:** `https://www.npmjs.com/package/create-git`
+
+## Module Description
+> A generator to initialize a project with git.
+
```

Comment:
These details are unnecessary for disclosure

Reply:
Removed them :)